keyboards/keyprez/rhino/keymaps/default/keymap.c

```diff
@@ -0,0 +1,124 @@
+                                                                                                                            KC_APP,
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,   KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_DEL,
+        _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,  _______, _______, _______, _______, _______, _______, KC_INS,
+        _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12, KC_NUHS, KC_NUBS, _______, _______, _______, KC_BSLS, _______,
+                 _______, _______, _______, _______,     _______,           _______,    KC_HOME, KC_PGUP, KC_PGDOWN, KC_END
+    ),
+
+    /* RAISE Layer
+     * .-----------------------------------------------------------------------------------------------------------------------------.
+     * |                                                                                                                     KC_APP |
+     * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------|
+     * | ~      | !      | @      | #      | $      | %      | ^      | &      | *      | (      | )      | -      | =      | |      |
+     * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+--------|
+     * |        | F1     | F2     | F3     | F4     | F5     | F6     | -      | =      | {      | }      |        |        | DEL    |
+     * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
+     * |        | F7     | F8     | F9     | F10    | F11    | F12    | ISO ~  | ISO |  |        |        |        |        | INSERT |
+     * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+-----------------+--------+--------|
+     * |        |        |        |        |        |                 |                 |        |        |        |        |        |
+     * '-----------------------------------------------------------------------------------------------------------------------------'
+     */
+    [_RAISE] = LAYOUT_ortho_2x2u(
+                                                                                                                             KC_APP,
+        KC_TILD, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR,    KC_ASTR,    KC_LPRN, KC_RPRN, KC_MINS, KC_EQL,  KC_PIPE,
+        _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_UNDS,    KC_PLUS,    KC_LCBR, KC_RCBR, _______, _______, KC_DEL,
+        _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  S(KC_NUHS), S(KC_NUBS), _______, _______, _______, _______, KC_INS,
+                 _______, _______, _______, _______,     _______,           _______,    _______, _______, _______, _______
+    ),
+};
+
+const uint8_t music_map[MATRIX_ROWS][MATRIX_COLS] = LAYOUT_ortho_2x2u(
```

Comment:
Any reason this isn't in the keyboard's C file?

Reply:
In the [docs ](https://beta.docs.qmk.fm/using-qmk/hardware-features/feature_audio), it is written that you should use keymap.c. I figured it also made sense because the `LAYOUT_...` is different for each of the keymaps. Which documentation should I look at to implement it like you are suggesting?

Reply:
If you're added the same config to each keymap, then it would make more sense to add it to the keyboard's C file instead, but set it with `__attribute__((weak))`, so that users can override it.  

This way, it still works if compiled on the QMK Configurator, too.